extra/entrypoint.sh

```diff
@@ -0,0 +1,13 @@
+#!/usr/bin/env sh
+
+set -e
+
+files_ownership () {
+    chown -hRc "${PUID=1000}":"${PGID=1000}" /app/data
```

Comment:
I thought for using default value sin bash the syntax was `${PUID:-1000}`

Reply:
Both notations [are correct](https://aavtech.site/2020/08/bash-using-or-setting-a-default-value-for-null-or-unset-variables/), depending on what you are going to archive.
In my case, using `${PUID:-1000}`
```
=> Performing startup jobs and maintenance tasks
+ echo ==> Performing startup jobs and maintenance tasks
+ files_ownership
+ chown -hRc -1000:-1000 /app/data
chown: invalid option -- '1'
Try 'chown --help' for more information.
+ echo ==> Performing startup jobs and maintenance tasks
+ files_ownership
==> Performing startup jobs and maintenance tasks
+ chown -hRc -1000:-1000 /app/data
chown: invalid option -- '1'
Try 'chown --help' for more information.
==> Performing startup jobs and maintenance tasks
+ echo ==> Performing startup jobs and maintenance tasks
+ files_ownership
+ chown -hRc -1000:-1000 /app/data
chown: invalid option -- '1'
Try 'chown --help' for more information.
==> Performing startup jobs and maintenance tasks
+ echo ==> Performing startup jobs and maintenance tasks
+ files_ownership
+ chown -hRc -1000:-1000 /app/data
chown: invalid option -- '1'
Try 'chown --help' for more information.
==> Performing startup jobs and maintenance tasks
+ echo ==> Performing startup jobs and maintenance tasks
+ files_ownership
+ chown -hRc -1000:-1000 /app/data
```
I can only guess that not declared environment variable in Docker is treated as unset, so it requires `${PUID:1000}` here.